README.md

```diff
@@ -1 +1,54 @@
-# aws-dynamodb-table
\ No newline at end of file
+# aws-dynamodb-table
```

Comment:
This is the old README template. Is your mass file up to date or has the new README not been added @WillBeebe to mass?

Reply:
The README in the CLI is the old one. I'll update it with the one from the bulk update script: https://github.com/massdriver-cloud/bundle-github-repo-manager/blob/main/bulk_edit/README.md.tmpl

Reply:
Save can you drop the same one in here. I’m assuming the precommit is the old one as well.

Reply:
It definitely was.